lib/IDE/CodeCompletionResultPrinter.cpp

```diff
@@ -109,6 +109,9 @@ class AnnotatingResultPrinter {
     case ChunkKind::Attribute:
       printWithTag("attribute", C.getText());
       break;
+    // FIXME: GenericParameterName corresponds to a type declaration
+    // and must be annotated respectively.
+    // case ChunkKind::GenericParameterName:
     case ChunkKind::BaseName:
       printWithTag("name", C.getText());
       break;
```

Comment:
Do we happen to also have an annotation for *type declarations* vs "other declarations" (which is what `<name>` apparently corresponds to), as Xcode calls them in the theme settings?

Reply:
Currently no. For generic parameters, I think we can just use `typeid.user` for now.